openbb_platform/core/openbb_core/provider/standard_models/equity_historical.py

```diff
@@ -36,10 +36,16 @@ class EquityHistoricalQueryParams(QueryParams):
 
     @field_validator("symbol", mode="before", check_fields=False)
     @classmethod
-    def upper_symbol(cls, v: str) -> str:
-        """Convert symbol to uppercase."""
+    def to_upper(cls, v: str) -> str:
+        """Convert field to uppercase."""
         return v.upper()
 
+    @field_validator("interval", mode="before", check_fields=False)
```

Comment:
We have "1m" and "1M" for intervals, where upper case M = Month and lower case m = minute.

Reply:
Thanks for raising that, I'll remove interval